radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/ParticleObservation.java

```diff
@@ -0,0 +1,32 @@
+package com.radixdlt.client.core.ledger;
+
+import java.util.Objects;
+
+/**
+ * A helper wrapper class for transitioned particles which can also represent HEAD events
+ */
+public class ParticleObservation {
+	private final TransitionedParticle particle;
+
+	private ParticleObservation(TransitionedParticle particle) {
+		this.particle = particle;
+	}
+
+	public TransitionedParticle getParticle() {
+		return particle;
+	}
+
+	public boolean isHead() {
+		return this.particle == null;
+	}
+
+	public static ParticleObservation head() {
```

Comment:
Could be a constant i.e.
`public static final ParticleObservation HEAD = new ParticleObservation(null)`